src/main/java/tasks/Palindrome.java

```diff
@@ -1,23 +1,23 @@
 package tasks;
 
-import utils.FileUtils;
-
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 
 public class Palindrome {
 
+    private TaskCore taskCore;
```

Comment:
please extent from TaskCore rather than create in all classes the TaskCore object.